src/pages/community/meet-the-team.mdx

```diff
@@ -50,6 +50,7 @@ have a track record of commitment to the long-term vitality of the project.
 
 | **NAME**                | **PROUDEST CONTRIBUTION**                                                                       |
 | ----------------------- | ----------------------------------------------------------------------------------------------- |
+| **cburroughs**          |                                                                                                 |
```

Comment:
@cburroughs welcome! please feel free to offer a suggestion for how you'd like to be listed and your proudest contribution!

Reply:
Can I defer this and see if I ever get lockfile updates over the finish line? ;-)

Reply:
Sure thing 👍